int32 mask
---
string message
